README.md

```diff
@@ -1,6 +1,126 @@
+[![Build Status](https://dev.azure.com/cncf/strimzi/_apis/build/status/test-clients?branchName=main)](https://dev.azure.com/cncf/strimzi/_build/latest?definitionId=38&branchName=main)
 [![License](https://img.shields.io/badge/license-Apache--2.0-blue.svg)](http://www.apache.org/licenses/LICENSE-2.0)
 [![Twitter Follow](https://img.shields.io/twitter/follow/strimziio.svg?style=social&label=Follow&style=for-the-badge)](https://twitter.com/strimziio)
 
-# Strimzi Template repository
 
-This repository is currently empty
+# Strimzi Test Clients
+
+This repository contains HTTP and Apache Kafka clients used in Strimzi systemtests:
+
+* Kafka Producer which periodically produces messages into a topic
+* Kafka Streams application which reads messages from a topic, transforms them (reverses the message payload) and sends them to another topic
+* Kafka Consumer which is consuming messages from a topic
+* Kafka Admin which allows user to alter/create/remove topics via Admin API
+* HTTP Producer which produces messages to a topic using host and port
+* HTTP Consumer which consumes messages from a topic and endpoint
+
+All clients are used as K8s `Job` - example files are present in [examples folder](examples).
+Logging configuration can be found in the `log4j2.properties` file in each module separately.
+
+## Build
+
+The pre-built images are available on our [Quay.io](https://quay.io/organization/strimzi-test-clients).
+But if you want to do any modifications to the clients, you will need to build your own versions.
+
+To build these examples you need some basic requirements.
+Make sure you have `make`, `docker`, `JDK 11` and `mvn` installed.
+By default, the Docker organization to which images are pushed is the one defined by the `USER` environment variable which is assigned to the `DOCKER_ORG` one.
+The organization can be changed exporting a different value for the `DOCKER_ORG` and it can also be the internal registry of an OpenShift running cluster.
```

Comment:
```suggestion
One can change the organization by exporting a different value for the `DOCKER_ORG`, and it can also be the internal registry of an OpenShift running cluster.
```